subsys/bluetooth/host/Kconfig

```
@@ -179,9 +179,10 @@ config BT_HOST_CRYPTO
 config BT_HOST_CRYPTO_PRNG
 	bool "Use PSA crypto API library for random number generation"
 	default y
-	select PSA_WANT_ALG_SHA_256
-	select PSA_WANT_KEY_TYPE_HMAC
-	select PSA_WANT_ALG_HMAC
+	# Fallback for boards that do not have an HW entropy generator (i.e. CSPRNG_ENABLED).
+	# This is meant to be used for tests only, not in production, because generated
+	# random values might be predictable/extremely weak.
```

It's not that they "might" be. It's that they are.
```suggestion
	# numbers are not random, which means not secure.
```